cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Wait for the database to be available...")
+        db_connect = None
+        while not db_connect:
+            try:
+                db_connect = connections["default"].ensure_connection()
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
+
+        self.stdout.write(self.style.SUCCESS("Database available!"))
```

Comment:
The messages being printed to stdout could be a bit more informative. For instance, when the database connection is successful, it would be helpful to print how many attempts were made or how long it took to establish the connection. This can provide useful debugging information if there are issues with the database connection in the future.